### PR TITLE
Add payint hosted payments page product

### PR DESCRIPTION
**Pull request by petar-barzev-cko**

### CODE REVIEW

File: spec/components/schemas/HostedPayments/HostedPaymentsRequest.yaml [hunk 1]
```diff
@@ -0,0 +1,152 @@
...
+            type: string
+            description: The international country calling code (https://docs.checkout.com/docs/country-codes). Required for some risk checks
+            pattern: \d{1,7}
+          number:
+            type: string
+            description: The phone number.
+            pattern: \d{6,25}
+  products:
+    type: array
+    description: Details about the order products
```
Comment (toby-knott-cko) [inline spec/components/schemas/HostedPayments/HostedPaymentsRequest.yaml:95]:
```suggestion
    description: Details about the products in the order
```

File: spec/components/schemas/HostedPayments/HostedPaymentsRequest.yaml [hunk 2]
```diff
@@ -0,0 +1,152 @@
...
+        type: object
+        additionalProperties: false
+        required:
+          - name
+          - quantity
+          - price
+        properties:
+          name:
+            type: string
+            description: Descriptive item name.
```
Comment (toby-knott-cko) [inline spec/components/schemas/HostedPayments/HostedPaymentsRequest.yaml:108]:
```suggestion
            description: Descriptive item name
```

File: spec/components/schemas/HostedPayments/HostedPaymentsRequest.yaml [hunk 3]
```diff
@@ -0,0 +1,152 @@
...
+          quantity:
+            type: integer
+            description: Non-negative. The item quantity.
```
Comment (toby-knott-cko) [inline spec/components/schemas/HostedPayments/HostedPaymentsRequest.yaml:111]:
```suggestion
            description: The item quantity. Non-negative
```

File: spec/components/schemas/HostedPayments/HostedPaymentsRequest.yaml [hunk 4]
```diff
@@ -0,0 +1,152 @@
...
+            minimum: 1
+          price:
+            type: integer
+            description: 'Minor units. Includes tax, excludes discount. (max value: 100000000)'
```
Comment (toby-knott-cko) [inline spec/components/schemas/HostedPayments/HostedPaymentsRequest.yaml:115]:
```suggestion
            description: 'Minor units. Includes tax, excludes discounts. (Max value: 100000000)'
```

File: spec/components/schemas/HostedPayments/HostedPaymentsRequest.yaml [hunk 5]
```diff
@@ -0,0 +1,152 @@
...
+            maximum: 100000000
+  success_url:
+    type: string
+    format: uri
+    description: For redirect payment methods, this overrides the default success redirect URL configured on your account
+    example: https://example.com/success
+  cancel_url:
+    type: string
+    format: uri
+    description: Direct customer to this URL if they cancel the payment
```
Comment (toby-knott-cko) [inline spec/components/schemas/HostedPayments/HostedPaymentsRequest.yaml:126]:
```suggestion
    description: The URL to which the customer should be directed if they cancel the payment
```

File: spec/swagger.yaml [hunk 6]
```diff
@@ -56,6 +56,8 @@ tags:
     description: Use events to monitor and get insights into your transactions.
   - name: Disputes
     description: Keep track of and act on your open disputes (also known as chargebacks) to submit the best possible response.
+  - name: Hosted Payments
```
Comment (toby-knott-cko) [inline spec/swagger.yaml:59]: I'm wondering where this should sit in the left menu. Maybe after the Payments section?

File: spec/components/schemas/HostedPayments/HostedPaymentsRequest.yaml [hunk 7]
```diff
@@ -0,0 +1,152 @@
...
+    example: https://example.com/cancel
+  failure_url:
+    type: string
+    format: uri
+    description: For redirect payment methods, this overrides the default failure redirect URL configured on your account
+    example: https://example.com/failure
+  metadata:
+    type: object
+    title: The Metadata Schema
```
[the remark this answers is not in the document]
Comment (toby-knott-cko) [reply spec/components/schemas/HostedPayments/HostedPaymentsRequest.yaml:135]: Wasn't sure if this is a correct description of the object, but just wanted to add one for consistency.

File: spec/swagger.yaml (hunk 6 above)
Comment (petar-barzev-cko) [reply spec/swagger.yaml]: Sure, I think that works and I have moved it there now.

**Review by toby-knott-cko — APPROVED** [review]

**Pull request merged**